task/sql-tasks.js

```diff
@@ -259,7 +424,19 @@ async function task_1_19(db) {
  *
  */
 async function task_1_20(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            Employees.EmployeeID as "EmployeeID",
+            CONCAT(Employees.FirstName, ' ', Employees.LastName) AS "Employee Full Name",
+            SUM(OrderDetails.UnitPrice * OrderDetails.Quantity) AS "Amount, $"
+        FROM Employees INNER JOIN Orders INNER JOIN OrderDetails
+            ON Employees.EmployeeID = Orders.EmployeeID
+            AND Orders.OrderID = OrderDetails.OrderID
+        GROUP BY Employees.EmployeeID
+        ORDER BY SUM(OrderDetails.UnitPrice * OrderDetails.Quantity) DESC

```

Comment:
why didn't you use short way (`ORDER BY 3 DESC`) of  ORDER BY clause  as in task1_4 ?